pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -74,12 +74,27 @@ func Apply(client clientset.Interface, target *deployments.Target) error {
 		return err
 	}
 
+	// Disable skuba-update.timer before upgrade
+	if skubaUpdateWasEnabled {
+		err = target.Apply(nil, "skuba-update-timer.disable")
+		if err != nil {
+			return err
+		}
+	}
+
```

Comment:
Note: there is _technically_ still a race here: Disabling skuba-update-timer will just ensure the service doesn't run. What if it ran before the `skuba node upgrade`, let a file in place where kured expects so. When skuba-update-timer gets disabled here, kured could possibly not have a lock yet at L86, and create one (because polling time) before L92 gets executed. That's because we don't check at L85 if a file marking a reboot already exists.

However, I think the proposed approach is currently the best:
- If we bail out at L85 by checking if a file marking the reboot is present, it would effectively prevent admins to upgrade their cluster at the same time a reboot is pending (therefore forcing a larger maintenance window). This could be overcomed by adding a flag to the `node upgrade` command (like `force`), but I have the impression it's over-engineering things.
- The likelihood of this happening is low.

If we really want to fix this, instead of overengineering, we could also (on top of this code) simply add a check if the reboot sentinel file exists, and just remove at the same time we are disabling the update timer at L79-84.